graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -54,8 +54,17 @@ def _estimate_filter_selectivity_of_equality(
     if _are_filter_fields_uniquely_indexed(filter_fields, unique_indexes):
         # TODO(evan): don't return a higher absolute selectivity than class counts.
         result_selectivity = Selectivity(kind=ABSOLUTE_SELECTIVITY, value=1.0)
+    else:
+        # Assumption: each distinct field value is equally common.
+        statistics_result = statistics.get_distinct_field_values_count(
+            location_name, filter_fields[0]
```

Comment:
This seems problematic. For example, I can't be sure since there's only a one-line docstring on this function, but I don't think we have a guarantee of ordering on the filter fields. This means that two runs with the the same input and statistics might produce two different estimates.

Reply:
You're right, this is very brittle. I'll use _combine_filter_selectivities instead,

> This seems problematic. For example, I can't be sure since there's only a one-line docstring on this function, but I don't think we have a guarantee of ordering on the filter fields. This means that two runs with the the same input and statistics might produce two different estimates.

You're right, this is very brittle. I'll use _combine_filter_selectivities instead,